uint8 vtol_state
uint8 landed_state